frontend/src/components/car-card/components/card-base/styles.module.scss

```diff
@@ -0,0 +1,108 @@
+.wrapper {
+  display: grid;
+  grid-template-rows: 18% minmax(1fr, 55%) 27%;
+  grid-template-columns: 1fr 80px;
+  box-sizing: border-box;
+  max-width: 400px;
+  min-height: 430px;
+  padding: 0 30px;
+  background-color: var(--white);
+  border-radius: 5px;
+}
+
+.carImage {
+  grid-column: span 2;
+  width: 100%;
+  border-radius: 5px;
+}
+
+.buttonsWrapper {
+  display: flex;
+  grid-column-start: 2;
+  justify-content: space-between;
+  justify-self: end;
+  align-items: center;
+  gap: 10px;
+}
+
+.carTitle {
+  display: flex;
+  gap: 10px;
+}
+
+.carLogo {
+  align-self: center;
+  width: 35px;
+  height: 35px;
+}
+
+.carName {
+  align-self: center;
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.likeButton {
+  width: 20px;
+  height: 20px;
+}
+
+.likeButton:hover {
+  cursor: pointer;
+}
+
+.compareButtonsWrapper {
+  margin-top: 4px;
+  margin-left: 10px;
+}
+
+.compareButton {
+  width: 20px;
+  height: 20px;
+}
+
+.compareButton:hover {
+  cursor: pointer;
+}
```

Comment:
Please use the SCSS nesting for the tightly couples classes. `compareButtonsWrapper` can include the `compareButton`, and the `compareButton` can include the `:hover`. Refer to this answer - https://stackoverflow.com/a/40905756/10963661